tokio-stream/src/stream_map.rs

```diff
@@ -364,11 +364,11 @@ impl<K, V> StreamMap<K, V> {
     /// # Examples
     ///
     /// ```
-    /// use std::collections::HashMap;
+    /// use tokio_stream::StreamMap;
```

Comment:
Fix import of `pending`.
```suggestion
    /// use tokio_stream::{StreamMap, pending};
```